charts/argo-cd/Chart.yaml

```diff
@@ -21,4 +21,4 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Fixed]: redis-ha.nameOverride / redis-ha.fullnameOverride breaks the ArgoCD helm chart"
+    - "[Updated]: Updated dex image 2.28.1 -> 2.30.0"
```

Comment:
`[Updated]` is not in our documentation. Please use one of whats documented :-) I think `[Security]` fits best

> Changelog
> We want to deliver transparent chart releases for our chart consumers. Therefore we require a changelog per new chart release.
> 
> Changes on a chart must be documented in a chart specific changelog in the Chart.yaml Annotation Section. For every new release the entire artifacthub.io/changes needs to be rewritten. Each change requires a new bullet point following the pattern - "[{type}]: {description}". You can use the following template:
> 
> name: argo-cd
> version: 3.4.1
> ...
> annotations:
>   artifacthub.io/changes: |
>     - "[Added]: Something New was added"
>     - "[Changed]: Changed Something within this chart"
>     - "[Changed]: Changed Something else within this chart"
>     - "[Deprecated]: Something deprecated"
>     - "[Removed]: Something was removed"
>     - "[Fixed]: Something was fixed"
>     - "[Security]": Some Security Patch was included"